contracts/SpokePool.sol

```diff
@@ -569,15 +570,15 @@ abstract contract SpokePool is
 
         // As a safety measure, prevent caller from inadvertently locking funds during exclusivity period
         //  by forcing them to specify an exclusive relayer if the exclusivity period
-        // is in the future. If this deadline is 0, then the exclusive relayer must also be address(0).
+        // is in the future. If this offset is 0, then the exclusive relayer must also be address(0).
         // @dev Checks if either are > 0 by bitwise or-ing.
-        if (uint256(uint160(exclusiveRelayer)) | exclusivityDeadline != 0) {
+        if (uint256(uint160(exclusiveRelayer)) | exclusivityDeadlineOffset != 0) {
             // Now that we know one is nonzero, we need to perform checks on each.
             // Check that exclusive relayer is nonzero.
             if (exclusiveRelayer == address(0)) revert InvalidExclusiveRelayer();
```

Comment:
ooc, do we need to retain this check? It prevents a depositor from setting 0x0 as the exclusiveRelayer, but it doesn't prevent the rest of the invalid (i.e. non-relayer) address space from being used. I think the main function would be to catch depositors accidentally setting this field, but that seems like an error that an integrator would quickly identify and fix because none of their testing would succeed.

```suggestion
            // Now that we know one is nonzero, we need to perform checks on each.
            // Check that exclusive relayer is nonzero.
            if (exclusiveRelayer == address(0)) revert InvalidExclusiveRelayer();
```

Reply:
Maybe not.

Nick and I spoke on this today, and we thought the only check that we may want to leave in is what's there now. Basically to not allow a user to set the exclusive relayer to 0 along with a nonzero exclusivity deadline, since that's clearly wrong. But we could also drop that one too just for simplicity, especially if we use 0x0 exclusive relayer as the on/off switch in the fill function.

Reply:
what do you think of the current check now @pxrl ? I'm not sure what your original push for was but the diff has updated

Reply:
This is simpler than what was there before :+1: I do still wonder if we could drop the check entirely, since the probability of funds locking funds is so low and so temporary.

Reply:
Yeah, the only thing this check protects against now is someone accidentally setting an exclusivity timestamp, but not a relayer. It's sort of just a bug signal since there is no reason anyone would want to do that.

I think removing is probably better -- saves gas, and this check doesn't really protect against anything.